pontoon/base/templates/landing.html

```diff
@@ -14,6 +14,7 @@
         <li><a href="{{ url('pontoon.search') }}">Terminology</a></li>
         <li><a href="https://developer.mozilla.org/en-US/docs/Localizing_with_Pontoon" target="_blank">Help</a></li>
         <li id="admin"{% if not perms.base.can_manage %} class="hidden"{% endif %}><a href="{{ url('pontoon.admin') }}">Admin</a></li>
+        <li id="sign-in" {% if user.is_authenticated() %} class="hidden" {% endif %}><a href="">Sign in</a></li>
```

Comment:
I would change this into something like this:
```
{% if user.is_authenticated() %}
<li id="sign-in"
{% else %}
<li id="signout"....
{% endif %}
```
It's better to not leave hidden things in markup if they aren't needed.